flake-registry.json

```diff
@@ -298,116 +33,6 @@
         "repo": "nixpkgs",
         "type": "github"
       }
-    },
-    {
-      "from": {
-        "id": "templates",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "NixOS",
-        "repo": "templates",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "patchelf",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "NixOS",
-        "repo": "patchelf",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "poetry2nix",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "poetry2nix",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "nix-serve",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "edolstra",
-        "repo": "nix-serve",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "nickel",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "tweag",
-        "repo": "nickel",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "bundlers",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "NixOS",
-        "repo": "bundlers",
-        "type": "github"
-      }
-    },
```

Comment:
Same argument for the bundlers as the templates

Reply:
Do bundlers actually work?

Whenever I see or hear them used, it is trying to fix issues or "does not work".